src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages_fr.properties

```diff
@@ -21,6 +21,7 @@ javadoc.paragraph.redundant.paragraph=Balise <p> redondante.
 javadoc.paragraph.tag.after=Une ligne vide doit être suivie par une balise <p> sur la ligne suivante.
 javadoc.parse.rule.error=Le commentaire Javadoc à la colonne {0} ne peut être analysé. Détails : {1} lors de l''analyse {2}
 javadoc.return.expected=Balise Javadoc @return manquante.
+javadoc.tag.line.before=Javadoc @-clause {0} doit être précédé d'une ligne vide.
```

Comment:
Double `''` required

Reply:
Ok. I also add double '' to the other message.properties files.